css/main.css

```diff
@@ -207,6 +207,25 @@ td, th {
   text-align: center;
 }
 
+/*===========================================================================
+   Style for submit button
+ ===========================================================================*/
+input[type=submit]
+{
+    border: 0.125em solid;
+    border-radius:0.25em;
+    width:5.25em;
+    padding: 0.3125em 0.625em;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 1.35em;
+    margin: 0.25em 0.125em;
+    -webkit-transition-duration: 0.4s; /* Safari */
+    transition-duration: 0.4s;
+    cursor: pointer;
+}
```

Comment:
Round the value of ems (in all the places _you_ gave them) to the first decimal or to the nearest integer